WordPressAuthenticator/Unified Auth/StoredCredentialsAuthenticator.swift

```diff
@@ -156,12 +156,63 @@ extension StoredCredentialsAuthenticator: LoginFacadeDelegate {
 
 @available(iOS 13, *)
 extension StoredCredentialsAuthenticator {
-    func presentLoginEpilogue(credentials: AuthenticatorCredentials) {
+    private func presentLoginEpilogue(credentials: AuthenticatorCredentials) {
         guard let navigationController = self.navigationController else {
             DDLogError("No navigation controller to present the login epilogue from")
             return
         }
         
         authenticationDelegate.presentLoginEpilogue(in: navigationController, for: credentials, onDismiss: {})
     }
+    
+    /// Presents the login email screen, displaying the specified error.  This is useful
+    /// for example for iCloud Keychain in the case where there's an error logging the user
+    /// in with the stored credentials for whatever reason.
+    ///
+    private func presentLoginEmailView(error: Error) {
+        guard let toVC = LoginEmailViewController.instantiate(from: .login) else {
+            DDLogError("Failed to navigate to LoginEmailVC from LoginPrologueVC")
+            return
+        }
+        
+        if let loginFields = loginFields {
+            toVC.loginFields = loginFields
+        }
```

Comment:
Why the check for `loginFields` instead of just setting it? We do `vc.loginFields = loginFields` all over, so this feels unnecessary. If there's a reason, could you add a comment?

Reply:
This is not a check, I'm unwrapping the optional `loginFields`.

The reason this is necessary is because the `toVC.loginFields` can't be optional, whereas loginFields is optional within `StoredCredentialsAuthenticator`.

Reply:
> toVC.loginFields can't be optional

Oh! Sorry! I missed that.